### PR TITLE
Fix bug in language prop and update dotcms library

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: components/layout/LocaleDropdown.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ Options.propTypes = {
     languages: PropTypes.arrayOf(
         PropTypes.shape({
             languageCode: PropTypes.string.isRequired,
-            id: PropTypes.string.isRequired,
+            id: PropTypes.number.isRequired,
```
Comment (freddyucv) [inline components/layout/LocaleDropdown.js]: any test?

Comment (fmontes) [reply components/layout/LocaleDropdown.js]: @alfredo-dotcms can we do tests on props?

Comment (alfredo-dotcms) [reply components/layout/LocaleDropdown.js]: @fmontes https://javascriptplayground.com/failing-tests-on-react-proptypes/

Comment (alfredo-dotcms) [reply components/layout/LocaleDropdown.js]: https://stackoverflow.com/a/31835256

Comment (alfredo-dotcms) [reply components/layout/LocaleDropdown.js]: https://medium.com/shark-bytes/type-checking-with-prop-types-in-jest-e0cd0dc92d5

I have not tried those solutions

**Pull request merged**